src/items/deleteItemDialog.component.tsx

```diff
@@ -16,35 +16,49 @@ import React from 'react';
 import { Link } from 'react-router';
 import { Item } from '../api/api.types';
 import { useDeleteItem } from '../api/items';
+import { useGetRules } from '../api/rules';
 import { useGetSystem } from '../api/systems';
 import handleIMS_APIError from '../handleIMS_APIError';
 
 export interface DeleteItemDialogProps {
   open: boolean;
   onClose: () => void;
-  item: Item | undefined;
+  item: Item;
   onChangeItem: (Item: Item | undefined) => void;
 }
 
 const DeleteItemDialog = (props: DeleteItemDialogProps) => {
   const { open, onClose, item, onChangeItem } = props;
 
-  const [error, setError] = React.useState(false);
   const [errorMessage, setErrorMessage] = React.useState<string | undefined>(
     undefined
   );
 
   const { data: systemData } = useGetSystem(item?.system_id);
+
+  // This should be a list of 1 rule
+  const { data: SelectedRule } = useGetRules(
```

Comment:
```suggestion
  const { data: deleteRules } = useGetRules(
```
Just a bit more specific, while it shouldnt be multiple maybe plural as well as its not really enforced here and should work regardless?

Reply:
`deletionRules` already exist as variable. This is all the delete rules. I'll change `SelectedRule` to `selectedRules` so it is in the right case (camelcase)

Reply:
I'll also update it to be plural everywhere

Reply:
Ah yeah I see it was on the next line 🤦‍♂️. Thanks.